### DISCUSSION TITLE
rendering of Code-Block and toTopButton broken on Server

### DISCUSSION THREAD

**Discussion by cnichte** [thuliteio/doks#1175] [General]
Hi, I have two problems that only occur on the Webserver, but not on the local development server  started with `npm run dev`: 

It worked for a while, but not since recently.

## Code-Block

If it runs in the local dev server, everything is fine:

![Bildschirmfoto 2024-01-13 um 16 50 19](https://github.com/gethyas/doks/assets/96977775/f6262aae-5258-44f4-a76b-a2bb3cd319c2)

If it is on a normal web server, it looks like this:

![Bildschirmfoto 2024-01-13 um 16 47 00](https://github.com/gethyas/doks/assets/96977775/80d4da43-0c52-4aaa-8205-ba546ee5fbb2)

## toTopButton

If it runs in the local dev server, everything is fine:

![Bildschirmfoto 2024-01-13 um 16 56 41](https://github.com/gethyas/doks/assets/96977775/55408833-d657-41d3-81bc-f03b28522d16)

its missing on Webserver also. 

Example: https://carsten-nichte.de/docs/einen-homeserver-einrichten/#emby--jellyfin

**Comment by jollydodo** [comment]
This is a purgecss issue. I had the same issue with the code blocks and it is is simply fixed by adding 'code' to your purgecss safelist located in config/postcss.config.js

**Reply by cnichte** [reply]
Thanks for the tip. I tried that, but it doesn't changed anything.

**Reply by h-enk** [reply]
Adding these lines should fix your code block issue: https://github.com/gethyas/doks/blob/main/config/postcss.config.js#L61-L63

**Comment by h-enk** [comment]
Adding `fade` to the `safelist` section should fix the `toTopButton` issue

**Reply by h-enk** [reply]
https://github.com/gethyas/doks/issues/1176

**Reply by cnichte** [reply]
Thanks, that fixed it finally.
